simp-beaker-helpers.gemspec

```diff
@@ -19,6 +19,7 @@ Gem::Specification.new do |s|
                  'issue_tracker' => 'https://simp-project.atlassian.net'
                }
   s.add_runtime_dependency 'beaker', '~> 3.14'
+  s.add_runtime_dependency 'beaker-puppet', '~> 0.8.0'
```

Comment:
Looks like beaker-puppet has had some puppet 5 related commits since 0.8.0

Reply:
This is the minimum viable version as far as I can tell.